app/src/main/java/kr/co/softcampus/sopt_assignment1/BindingAdapter.kt

```diff
@@ -0,0 +1,13 @@
+package kr.co.softcampus.sopt_assignment1
+
+import android.widget.ImageView
+import androidx.databinding.BindingAdapter
+
+object BindingAdapter {
+
+    @JvmStatic
+    @BindingAdapter("imgRes")
+    fun setImage(imageView: ImageView, resid: Int) {
+        imageView.setImageResource(resid)
+    }
```

Comment:
확장함수를 사용해서 이렇게도 작성할 수 있습니당 :) 
```suggestion
    fun ImageView.setImage(resid: Int) {
       setImageResource(resid)
    }
```